### PR TITLE
Implementation of new Photonuclear Parametrizations

### PR DESCRIPTION
**Pull request by asandrock**
New photonuclear parametrizations according to Block, Durand & Ha (2014) and Abt et al. (2017)

### CODE REVIEW

File: private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx [hunk 1]
```diff
@@ -616,4 +618,225 @@ double PhotoRenoSarcevicSu::FunctionToQ2Integral(double energy, double v, double
...
+    // differential cross section from Dutta et al.
+    // Phys. Rev. D 63 (2001), 094020
+    // eq. 3.11
+    // eq. 3.4
+    // \frac{d^2 \sigma}{dQ^2 dx} = \frac{4\pi\alpha^2}{Q^4} \frac{F_{2, Proton}}{v}
+    //     ( 1 - v + \frac{M x v}{2E}
+    //     - \frac{v^2}{2} (1 - \frac{2m_{particle}}{Q^2})
+    //          \frac{1 + \frac{4 M^2 x^2}{Q^2}}{1 + R})
+    double result = ME * RE / Q2;
+    result *= result * 4 * PI * structure_function_nucleus / v *
```
Comment (maxnoe) [inline private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx:749]: Is this really correct? `result *= result ... ` look looks suspicious.

Comment (sudojan) [reply private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx:749]: yes, that's quite often the way how things are squared in PROPOSAL.
It's probably not code-reader friendly

Comment (maxnoe) [reply private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx:749]: Maybe worth taking a look to implement a `pow(double, int)` ?

https://stackoverflow.com/a/13771544/3838691

Comment (maxnoe) [reply private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx:749]: Actually, all current compilers just replace `pow(x, 2)` with `x * x`: https://godbolt.org/z/a3c5sY

Comment (maxnoe) [reply private/PROPOSAL/crossection/parametrization/PhotoQ2Integration.cxx:749]: So the above is only beneficial for integer powers > 2

**Review by sudojan — APPROVED** [review]

**Comment by sudojan** [comment]
This will close issue #125

**Pull request merged**